appOPHD/States/MapViewStateTurn.cpp

```diff
@@ -414,37 +416,6 @@ void MapViewState::updateResources()
 }
 
 
-/**
- * Check for colony ship deorbiting; if any colonists are remaining, kill
- * them and reduce morale by an appropriate amount.
- */
-void MapViewState::checkColonyShip()
-{
-	if (mTurnCount == constants::ColonyShipOrbitTime)
-	{
```

Comment:
Or perhaps rather than an event, we could have continued to poll the colony ship status here, and handle the UI related portions of the crash event.

The updates to core game mechanics are of course better handled by `ColonyShip`.